src/messaging/tests/TestExchangeMgr.cpp

```diff
@@ -161,10 +162,10 @@ void CheckUmhRegistrationTest(nlTestSuite * inSuite, void * inContext)
     err = exchangeMgr.Init(&conn);
     NL_TEST_ASSERT(inSuite, err == CHIP_NO_ERROR);
 
-    err = exchangeMgr.RegisterUnsolicitedMessageHandler(0x0001, HanldeAllUnsolicitedMessage, nullptr);
+    err = exchangeMgr.RegisterUnsolicitedMessageHandler(0x0001, &gMockAppDelegate);
```

Comment:
thanks for your effort on this,  do you think you can add the test code on modified RegisterUnsolicitedMessageHandler
basically to show how to use RegisterUnsolicitedMessageHandler